snowdrop-bot/README.md

```diff
@@ -0,0 +1,47 @@
+
+# Introduction
+
+Ansible playbook to deploy the `snowdrop-bot` into k8s using [quay.io](https://quay.io/) registry.
```

Comment:
We dont deploy the application but the role defined part of this PR is about compile or package/build/push the image to the quay registry. Can you then review this description section please ?